src/coverlet.inproccollector/CoverletInProcDataCollector.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
```

Comment:
Why not just include this file in the `coverlet.collector` project? You could put it in an `InProcDataCollector` folder

Reply:
Yeah. That can be done.

Reply:
Good idea I agree!

Reply:
Yes. Done.